src/renderer/lib/hooks/useAccount.ts

```diff
@@ -34,10 +36,10 @@ export const useAccount = create(
 
         set({ loading: true });
         await fetchPlayKey()
-          .then((playKey) => set({ playKey }))
+          .then((playKey) => set({ playKey, isServerError: false }))
           .catch((err) => {
             console.warn("Error fetching play key: ", err);
-            set({ playKey: null });
+            set({ playKey: null, isServerError: true });
```

Comment:
Ok so we actually need to think of another way to do this. A user with a brand new account triggers this bit of code and thus they are never able to set their connect code.

Reply:
Huh interesting. In all likelihood I feel like an account without a play key should return a successful result with an empty play key, not an error. I can take a look tomorrow too.

Reply:
Addressed by https://github.com/project-slippi/slippi-launcher/pull/275/commits/ac47b418e12d4fc6cbacb68d6982f2ed3dfae9f4

The `connectCode` and `private` objects on `getUser` were null for a new user.